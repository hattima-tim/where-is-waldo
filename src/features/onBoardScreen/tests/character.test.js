import { render } from "@testing-library/react";
import Character from "../character";

test("Character component is rendered correctly", () => {
  const { container } = render(
    <Character
      difficultyLevel={"easy"}
      src={
        "https://res.cloudinary.com/du3oueesv/image/upload/v1672759163/Where%27s%20Waldo/cacodemon.1dad269c_ajnuga.png"
      }
      alt={"cacodemon"}
      characterName={"Cacodemon"}
      reference={"Doom"}
    />
  );

  expect(container).toMatchSnapshot();
});
