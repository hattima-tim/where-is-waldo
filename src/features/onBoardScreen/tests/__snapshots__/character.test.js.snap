// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component is rendered correctly 1`] = `
<div>
  <div
    class="flex w-full flex-col"
  >
    <p
      class="self-end"
    >
      easy
    </p>
    <div
      class="flex gap-4"
    >
      <img
        alt="cacodemon"
        class="w-8"
        src="https://res.cloudinary.com/du3oueesv/image/upload/v1672759163/Where%27s%20Waldo/cacodemon.1dad269c_ajnuga.png"
      />
      <div>
        <h4>
          Cacodemon
        </h4>
        <p>
          Doom
        </p>
      </div>
    </div>
  </div>
</div>
`;
